ludwig/schema/combiners/tabnet.py

```diff
@@ -35,34 +37,34 @@ def module_name():
 
     output_size: int = schema_utils.PositiveInteger(
         default=128,
-        description="Output size of a fully connected layer. `N_d` in the paper",
+        description="Output size of a fully connected layer. `N_d` in the paper.",
         parameter_metadata=COMBINER_METADATA["TabNetCombiner"]["output_size"],
     )
 
     num_steps: int = schema_utils.NonNegativeInteger(
         default=3,
         description="Number of steps / repetitions of the the attentive transformer and feature transformer "
-        "computations. `N_steps` in the paper ",
+        "computations. `N_steps` in the paper.",
```

Comment:
Nit: I know it's a seminal paper but perhaps we should say `(Arik and Pfister, 2019)` instead of literally `the paper`.

Reply:
Good point, I agree that's. good change.